sunpy/visualization/mapcubeanimator.py

```diff
@@ -33,15 +33,26 @@ class MapCubeAnimator(imageanimator.BaseFuncAnimator):
     colorbar: bool
         Plot colorbar
 
+    plot_function: function
+        A function to call when each frame is plotted, the function must have
```

Comment:
Map and frame are interchangeable, so I will probably change it. An example is a good idea. This whole PR needs some more docs, I wanted to throw it out there to get some feedback on it.